### ISSUE TITLE
3/16 + Test Database Connectivity + 1

### ISSUE THREAD

**Issue by anazariz** [lzim/teampsd#1186]
**1. Add description:** 

Please test end-user database connectivity by using the workbook in our team's folder / quant folder / ash work area / LSV. Open the (only) workbook in that folder and click on the 5 buttons in it. You should get either an error, which represents the issue that our end-users potentially have (with missing database privileges) or an OK, which represents folks in our team who have database access privileges, which is the desirable state for our end-users as well.
 
**Please note that actual folder and file names have not been used here.** and loose references assume that you have infrastructure knowledge.   

<< Paste screenshots here>>
 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
viii. **Releases** - assign to key team release grouped with other relevant issues  

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by lzim** [comment]
@anazariz my test worked smoothly & I emailed you the results!

Both Jenn & Rita should provide good tests.
Couldn’t all our facilitators try thIs test? Specifically, test the LSV button?

**Comment by anazariz** [comment]
@lzim -  Yes they (facilitators) could. 

If you've requested DB access for Jenn and Rita in the past, then we need to go directly to users who potentially have no DB access.

**Comment by lzim** [comment]
> @lzim - Yes they (facilitators) could.
> 
> If you've requested DB access for Jenn and Rita in the past, then we need to go directly to users who potentially have no DB access.

@anazariz 
Right: Jenn & Rita have never received any type of database access, which is why they are provide a good test.

The facilitators do have other DB access. But if the question is whether they have LSV access, and this is our first test of LSV, then they also add value as testers.
